src/AppInstallerRepositoryCore/Microsoft/Schema/Portable_1_0/PortableTable.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PortableTable.h"
+#include "SQLiteStatementBuilder.h"
+#include "Microsoft/Schema/IPortableIndex.h"
+
+namespace AppInstaller::Repository::Microsoft::Schema::Portable_V1_0
+{
+    using namespace std::string_view_literals;
+    static constexpr std::string_view s_PortableTable_Table_Name = "portable"sv;
+    static constexpr std::string_view s_PortableTable_Index_Separator = "_"sv;
+    static constexpr std::string_view s_PortableTable_Index_Suffix = "_index"sv;

```

Comment:
these are not used, I'm not sure if index is needed, probably not because the entries count are expected to be low